src/SearchInput/SearchInput.tsx

```diff
@@ -23,11 +25,14 @@ export interface SearchInputProps extends CommonFieldProps {
   placeholder?: string
   searchList: SearchInputItem[]
   onFound: (element: string) => void
+  onNotFound?: () => void
+  notFoundComponent?: ReactNode
```

Comment:
Would we be able to add JSDocs to these new props? Adding JSDocs above the props will surface the doc on tooltips and should also display on storybook.
e.g.
```suggestion
  /**
  * Callback that runs once when search finds zero results
  */
  onNotFound?: () => void
  notFoundComponent?: ReactNode
```

Reply:
I've added the JSDocs but they're not rendering and I have a feeling it's to do with the use of `forwardRef` for our form components. Somewhat related, I had to set the all of the args within storybook as they weren't properly being created due to [this unresolved bug](https://github.com/storybookjs/storybook/issues/21686)